### ISSUE TITLE
auth_opt_log_level does not work

### ISSUE THREAD

**Issue by eydonmlau** [iegomez/mosquitto-go-auth#273]
Hi, When I set auth_opt_log_level in conf such like auth_opt_log_level debug, it doesn't work, it use the default info level. 
Is it need to add `log.SetLevel(authPlugin.logLevel)` at https://github.com/iegomez/mosquitto-go-auth/blob/9e9d90b676c97ec069bd39d96fa2235269f83932/go-auth.go#L86

**Comment by iegomez** [comment]
Yeah, that sounds like a regression when refactoring some backends to receive the log level.
I don't have the time right now, but I'll look into fixing it when I do.

**Comment by iegomez** [comment]
@eydonmlau  I checked and though the level is not set there, the `logLevel` option is passed to every backend initializer.
Now, there are some odd things I found:
- For some reason GRPC backend is not setting it, but it's only logging warnings and errors, so the default info level should be enough.
- JWT'S JS and Remote checkers don't expect it, but the general JWT backend initializer is setting the passed level that should cover it: https://github.com/iegomez/mosquitto-go-auth/blob/master/backends/jwt.go#L41
- Neither go-auth.go nor backends.go set it, but since any debug logging seems to be done after backends have been initialized (e.g., when checking auth or acl), the level should already be set.

Now, I don't remember what was the rationale behind passing a level to each backend, maybe it had to do with tests or something like that. Since there's a single option, I don't see why we need it. But, as I said, it should still work so I'm curious what's your setup and which logs are you missing.
